tests/modules/individuals/conftest.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=missing-docstring,redefined-outer-name
+import pytest
+
+from app.modules.encounters.models import Encounter
+from app.modules.individuals.models import Individual
+from app.modules.users.models import User
+
+
+def _user():
+    _user = User(email='test@user', password='testuser', full_name='Test User')
+    return _user
+
+
+@pytest.fixture()
+def basic_user():
+    return _user()
+
+
+@pytest.fixture()
```

Comment:
Good idea but dummy encounters and individuals are something that is going to be needed for many tests. 
Put them into tests/conftest.py instead?